packages/http/tools/e2e-tests/src/warrior/step-definitions/thenDefinitions.ts

```diff
@@ -17,31 +31,41 @@ async function thenResponseContainsTheCorrectUrlParametersByName(
 
   const warriorWithId: WarriorWithId = (await response.json()) as WarriorWithId;
 
-  assert.ok(warriorWithId.id === '123');
+  assert(warriorWithId.id === '123');
```

Comment:
We should compare it the request param. We can probably use the world to accomplish this

Reply:
I think this is not related to this PR. I will submit a fix in another PR